commands/show/cluster/command.go

```diff
@@ -347,6 +347,25 @@ func printResult(cmd *cobra.Command, cmdLineArgs []string) {
 	clusterDetailsV4, clusterDetailsV5, nodePools, clusterStatus, credentialDetails, err := getClusterDetails(args)
 	if err != nil {
 		errors.HandleCommonErrors(err)
+
+		headline := ""
+		subtext := ""
+
+		switch {
+		case errors.IsClusterNotFoundError(err):
+			headline = "Cluster not found"
+			subtext = fmt.Sprintf("Either there is no cluster with ID '%s', or you have no access to it.\n", args.clusterID)
```

Comment:
```suggestion
			subtext = fmt.Sprintf("No cluster with ID '%s' exists.\n", args.clusterID)
```
While the security aspect here is user friendly, I think that in general it's not taken into account and when person doesn't have rights to some information, it mostly by intention.

Reply:
We do it this way in most 404 cases, so I'd pass the suggestion in favour of consistency.